src/wizard/wizard.scss

```diff
@@ -2,164 +2,142 @@
         }
-        font-weight: $iui-font-weight-semibold;
-      }
-
-      > .iui-wizards-step-circle {
-        @include themed {
-          color: t(iui-color-foreground-accessory);
-          border: 1px solid t(iui-color-background-positive);
-          background-color: t(iui-color-background-positive);
+  
+        .iui-circle {
+          font-weight: $iui-font-weight-semibold;
+          @include themed {
+            color: t(iui-color-foreground-accessory);
+            border-color: t(iui-color-background-positive);
+            background-color: t(iui-color-background-positive);
+          }
+        }
+  
+        .iui-before {
+          @include themed {
+            background-color: t(iui-color-foreground-positive);
+          }
         }
-        font-weight: $iui-font-weight-semibold;
-      }
 
-      > .iui-wizards-step-track-before {
-        @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
+        .iui-track:first-child {
+          @include themed {
+            background-color: t(iui-color-foreground-positive);
+          }
         }
       }
-    }
-
-    > .iui-wizards-step-completed {
-      &.iui-clickable {
-        > .iui-wizards-step-circle {
+  
+      &.iui-completed {
+        &.iui-clickable .iui-circle {
           @include iui-focus(iui-focus-positive-box-shadow);
-          cursor: pointer;
-          transition: background-color $iui-speed-fast ease-out;
 
           &:hover {
+            cursor: pointer;
             @include themed {
-              color: t(iui-color-foreground-accessory);
-              border: 1px solid t(iui-color-background-positive);
               background-color: t(iui-color-background-positive);
+              border-color: t(iui-color-background-positive);
+              color: t(iui-color-foreground-accessory);
             }
           }
         }
-      }
-
-      > .iui-wizards-step-title {
-        @include themed {
-          color: t(iui-color-foreground-positive);
+  
+        .iui-step-name {
+          @include themed {
+            color: t(iui-color-foreground-positive);
+          }
         }
-      }
-
-      > .iui-wizards-step-circle {
-        @include themed {
-          color: t(iui-color-foreground-positive);
-          border: 1px solid t(iui-color-foreground-positive);
+  
+        .iui-circle {
+          @include themed {
+            color: t(iui-color-foreground-positive);
+            border-color: t(iui-color-foreground-positive);
+          }
+        }
+  
+        .iui-track {
+          @include themed {
+            background-color: t(iui-color-foreground-positive);
+          }
         }
       }
 
-      > .iui-wizards-step-track {
-        @include themed {
-          background-color: t(iui-color-foreground-positive) !important;
-        }
+      &:first-child .iui-track:first-child,
+      &:last-child .iui-track:last-child {
+        background-color: transparent;
```

Comment:
Well this isn't very nice
![image](https://user-images.githubusercontent.com/9084735/119188995-7435e480-ba49-11eb-9ff4-96dd972490af.png)

I would think those tracks should just not be present in the html. But then it messes up the centering.

What if we move those tracks outside the steps? I think it would even simplify the css and you wouldn't need the `.iui-idk` class 😄

Reply:
I don't think this is that bad.  It helps keep each step set to an equal width no matter the text underneath.  We want to avoid situations like this where a very long label VS short label off-centers the entire wizard.
![Screen Shot 2021-05-25 at 11 40 23 AM](https://user-images.githubusercontent.com/849817/119527102-01c94b00-bd4e-11eb-8340-5c21cdb22cc2.png)

1 option I thought of but think is not a good idea is having the text labels be within their own flex box.  That way the first / last step's circle could go all the way to the edge.  However, then you'll end up with something like this.
![Screen Shot 2021-05-25 at 11 47 36 AM](https://user-images.githubusercontent.com/849817/119528225-02aeac80-bd4f-11eb-8ff1-4d491ef5ea8a.png)

BTW the wizard refactor inspiration came from [here](https://csslayout.io/patterns/wizard).

Reply:
But when I start to think, it really simplifies React implementation 😃